packages/components/src/Disclosure/Disclosure.mdx

```diff
@@ -0,0 +1,114 @@
+---
+name: Disclosure
+menu: Components
+route: /components/disclosure
+showDirectoryLink: true
+---
+
+import { Playground, Props } from "docz";
+import { Content } from "@jobber/components/Content";
+import { Disclosure } from ".";
+
+# Disclosure
+
+```ts
+import { Disclosure } from "@jobber/components/Disclosure";
+```
+
+<Playground>
+  <Disclosure title="Open to see more content organa palpatine lobot jawa kamino.">
+    <Content>
+      <p>
```

Comment:
If we can import `Text` it would be nice to show this example built with all Atlantis!